src/shell/lisa_shell

```diff
@@ -373,6 +373,69 @@ echo
 
 
 ################################################################################
+# LISA Workloads utility functions
+################################################################################
+
+function lisa-wltest-init {
+export LISA_VENV="$LISA_HOME/tools/wltests/wa3.venv"
+export LISA_WAUTO="$LISA_HOME/tools/wltests/wlauto"
+export WA_USER_DIRECTORY="$LISA_HOME/tools/wltests/.wlauto"
+
+# If the python virtual env existis:
+# let's assume everithing has been already setup and we are ready to go
+[ ! -d $LISA_VENV ] || return
+
+# Check for require dependencies
+which virtualenv &>/dev/null
+if [ $? -ne 0 ]; then
+	echo -e "\n\nERROR: No python's virtualenv support found!"
+	echo -e "Please 'pip intall virtualenv' before running this command.\n\n"
+	return -1
+fi
+
+# Create and activate a python's virtual environment to be used for the
+# installation of the required version of external libraries and tools
+mkdir -p $LISA_VENV
```

Comment:
I think this mkdir is done automatically by virtualenv?

Reply:
likely...